build.gradle

```diff
@@ -116,7 +116,7 @@ subprojects {
         testCompile "io.circe:circe-parser_$scalaVersion:$circeVersion"
 
         testCompile 'junit:junit:4.12'
-        testCompile "org.scalatest:scalatest_$scalaVersion:3.0.7"
+        testCompile "org.scalatest:scalatest_$scalaVersion:3.0.8"
```

Comment:
There's actually version `3.1.0` now.

Reply:
There are also much newer versions of other libraries too. The main point of this PR is to update the `ampq-client`, so I wanted to update just minor versions of the rest of runtime libraries.